src/mlang/mpp_ir/mpp_ir_to_bir.ml

```diff
@@ -177,9 +178,44 @@ let translate_m_code (m_program : Mir_interface.full_program)
         let var = Mir.VariableDict.find vid m_program.program.program_vars in
         match vdef.var_definition with
         | InputVar -> None
-        | _ ->
+        | TableVar (i, idx_def) ->
```

Comment:
Could we use `Mir.map_var_def_var` to factor the code in here?

Reply:
Indeed. It's done.